articles/how-to-choose-levels-of-logging/index.md

```diff
@@ -0,0 +1,63 @@
+
+### Logging levels explained
+
+The most common logging levels include FATAL, ERROR, WARN, INFO, DEBUG, TRACE, ALL, and OFF. Some of them are important, others less important, while others are meta-considerations. So when logging from applications, you can follow the common logging levels as explained below.
+
+#### FATAL
+
+FATAL represents that the application is about to stop a serious problem or some corruption. The FATAL level of logging shows that the application&#39;s situation is catastrophic such that an important function is not working. You can use this FATAL log level if the application is unable to connect to the data store.
+
+#### ERROR
+
+Unlike FATAL, your application is not aborting; instead, there is just an inability to access a service or a file. This ERROR shows a failure in your application of something important going on. This log level is used when a severe issue is stopping functions within the application from properly functioning. Most times, the application will continue to run, but eventually, the will require to be addressed.
+
+#### WARN
+
+The WARN log level is used when you have detected an unexpected application problem. You may not notice any harm to your application at this point. And you are not quite sure whether the problem will happen again or whether the problem will remain. This issue is usually a situation that stops specific processes from happening. But it does not mean that the application has been harmed. In fact, the code continues to work as usual. You should eventually check these warnings just in case the problem recurs.
+
+#### INFO
+
+INFO messages are like the normal behavior of applications. They state what happened, for example, if a particular service stopped or started or whether you added something to the database. These entries are nothing to worry about during usual operations. The information logged using the INFO log is usually informative, and it does not necessarily require you to follow up on it.
+
+#### DEBUG
+
+With DEBUG, you are giving diagnostic information in a detailed manner. It is verbose and has more information than you would need when using the application. DEBUG logging level is used to fetch information needed to diagnose, troubleshoot, or test an application. This ensures the smooth running of an application.
+
+#### TRACE
+
+TRACE log level captures all details about the behavior of the application. It is mostly diagnostic and is more granular and finer than DEBUG log level. This log level is used in situations where one needs to see what happens in your application or what happens in the third-party library used. You can use the TRACE log level to query parameters in the code or interpret the algorithm&#39;s steps.
+
+#### ALL
+
+This log level logs any logging levels that are defined. It logs everything and includes custom logging levels as well. It is merely the unification of all other logging levels where the whole lot of loggings will be logged.
+
```

Comment:
Please rewrite this last sentence using simpler and commonly used terms. Ie. In "merely the unification" and "whole lot of loggings will be logged".